deps/llvm.mk

```diff
@@ -173,7 +173,7 @@ endif
 
 ifneq ($(LLVM_VER),svn)
 ifeq (,$(findstring rc,$(LLVM_VER)))
-ifeq ($(shell [ $(LLVM_VER_MAJ) -ge 8 -a $(LLVM_VER) != 8.0.0 ]; echo $$?),0)
+ifeq ($(shell [ x"$(LLVM_VER)" = x"8.0.1" ]; echo $$?),0)
```

Comment:
Won't this conditional fail for LLVM 9 now?

Reply:
yeah, turns out llvm 8.0.1 was the only release they decided not to upload to releases.llvm.org and use a github url instead. I mistakenly assumed that mean they were switching to that.

Reply:
:/